### PR TITLE
Added mouseover tooltips (Chart buttons)

### PR DESCRIPTION
**Pull request by ay-bh**
This PR adds mouseover tooltips to the Reset, Zoom, and Pan buttons on the right side of the charts. It resolves https://github.com/jlab-sensing/DirtViz/issues/156.

#### Changes Made:
- Added MUI `Tooltip` for buttons in `ChartWrapper.jsx`.
- Resolved naming conflict by aliasing `chart.js` Tooltip as `chartTooltip`.

#### Testing Performed:

Manual testing was done to ensure the chart.js Tooltip functionality remains unaffected and continues to work well alongside the newly added button tooltips.

### CODE REVIEW

File: frontend/src/charts/ChartWrapper.jsx
Comment (aaron-wu1) [inline frontend/src/charts/ChartWrapper.jsx:1]: could the placement of the tooltips be under the button rather than to the right?

**Review by aaron-wu1 — COMMENTED** [review]
Looks good, wondering if the tooltip could be positioned to the bottom rather to the right. Currently, due to the constraints of the browser window, the tooltips flip placement depending if it's the two right most charts or the two left most charts

File: frontend/src/charts/ChartWrapper.jsx
Comment (ay-bh) [reply frontend/src/charts/ChartWrapper.jsx:1]: @aaron-wu1 I tried that first, but the bottom placement tooltip covered the button below. If that's fine, I can make the changes. Please take a look at the screenshot.
image

Comment (aaron-wu1) [reply frontend/src/charts/ChartWrapper.jsx:1]: That should be fine, but make the offset a bit closer to the button and disable the interaction on the tooltip. So if you hover over the tooltip itself, it doesn't stay enabled.

Comment (ay-bh) [reply frontend/src/charts/ChartWrapper.jsx:1]: @aaron-wu1 Done. Please check now.

**Comment by ay-bh** [comment]
@aaron-wu1 I have synced fork with main. Please check now.

**Comment by aaron-wu1** [comment]
lgtm!

**Pull request merged**